src/widgets/tables/data_preview_screen.py

```diff
@@ -1,41 +1,110 @@
-        self.data_table.horizontalHeader().sectionDoubleClicked.connect(self.handleHeaderClick)
+        self.data_table.horizontalHeader().sectionDoubleClicked.connect(handle_header_click)
+        self.data_table.doubleClicked.connect(handle_cell_click)
         self.layout.addWidget(self.data_table)
 
-    def handleHeaderClick(self, index):
+    def reduction_header_click(self, index):
         newHeader, ok = QInputDialog.getText(self,
-                                                   'Change header label for column %d' % index,
-                                                   'Header:',
-                                                   QLineEdit.Normal,
-                                                   "")
+                                             f'Change header label for column {index}',
+                                             'Header:',
+                                             QLineEdit.Normal,
+                                             "")
         if ok:
             self.engine.rename_column(index, newHeader)
             self.data_table.setModel(QtTable(self.engine.state.imported_data))
 
+    def estimation_header_click(self, index):
+        newValue, ok = QInputDialog.getText(self,
+                                            'Default value for column %d:' % index,
+                                            'Value:',
+                                            QLineEdit.Normal,
+                                            "")
+        if ok:
+            column = self.engine.state.imported_data.iloc[:, index]
+            newValue = self.cast_input_type(column.dtype, newValue)
+            if newValue is not None:
+                column.fillna(newValue, inplace=True)
+            self.data_table.setModel(QtTable(self.engine.state.imported_data))
+
+    def estimation_cell_click(self):
+        cell = self.data_table.selectionModel().selectedIndexes()[0]
+        row, col = cell.row(), cell.column()
+        newValue, ok = QInputDialog.getText(self,
+                                            f'Default value for cell ({row}, {col}):',
+                                            'Value:',
+                                            QLineEdit.Normal,
+                                            "")
+        if ok:
+            column = self.engine.state.imported_data.iloc[:, col]
+            newValue = self.cast_input_type(column.dtype, newValue)
+            if newValue is not None:
+                self.engine.state.imported_data.iloc[row, col] = newValue
+            self.data_table.setModel(QtTable(self.engine.state.imported_data))
+
+    @staticmethod
+    def cast_input_type(data_type, value):
+        try:
+            match data_type:
+                case 'int32' | 'int64':
+                    return int(value)
+                case 'float64':
+                    return float(value)
+                case 'datetime64[ns]':
+                    return strptime(value)
```

Comment:
Are we sure that these are all good type?

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.dtypes.html - took types from the docs and checked experimentally using our database, it should work in all cases. int32 may be redundant, because int32 from mongo is cast to int64, but I would prefer to keep it anyway, because maybe on some data it can be cast to int32